schemas/individual_response/definitions.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$id": "https://eq.ons.gov.uk/individual_response/definitions.json",
+  "individual_response": {
+    "type": "object",
+    "properties": {
+      "enabled": {
+        "type": "boolean",
+        "description": "Controls whether the individual response definition can be shown"
+      },
+      "for_list": {
+        "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
+        "description": "The list to use when determining whether the individual response definition can be shown"
+      },
+      "individual_section_id": {
+        "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
+        "description": "The id of the individual section for which an individual response can be requested"
+      }
+    },
+    "additionalProperties": false,
+    "required": ["enabled", "for_list", "individual_section_id"]
```

Comment:
The required list doesn't make sense, if the `enabled` property was set to `false` wouldn't we still have to provide `for_list` and `individual_section_id`?

As the `individual_response` property is optional, we could just drop the enabled property.